Gruntfile.coffee

```diff
@@ -386,7 +386,7 @@ module.exports = (grunt) ->
 			i18n:
 				options:
 					process: ( src, filepath ) ->
-						lang = filepath.replace "dist/js/i18n/", ""
+						lang = filepath.replace "dist/wet-boew/js/i18n/", ""
```

Comment:
Unfortunately no. It doesn't resolve <code>&lt;%=coreDist %&gt;</code>, it just leaves that as a string.